src/pages/cart.js

```diff
@@ -15,6 +18,7 @@ export const Cart = () => {
     cartFetchStatus: state.cart.status,
     productsFetchStatus: state.productList.status,
   }))
+  const [lastQuantityValue, rememberLastQuantityValue] = useState(0)
```

Comment:
what is it used for?

Reply:
we have to store the previous value of cart entry if it is going to be set as 0
in order to revert it back in case if user won't remove that entry